### PR TITLE
Adds Offship Lathe Materials

### PR DESCRIPTION
**Pull request by courierbravo**
Adds aluminum, lead, and plastic to every ship with a lathe.

### CODE REVIEW

**Comment by courierbravo** [comment]
!review

**Comment by courierbravo** [comment]
note: Changes to the peacekeeper ship are for the kitchen. I was going to make this PR a bit broader in scope and just fix things as I found them. But I feel like if I'm gonna do those kinds of things, they should be their own separate PR or series of PR's

File: html/changelogs/CourierBravo - Lathe_Mats.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#   config
+#     - (makes a change to the config files)
+#   admin
+#     - (makes changes to administrator tools)
+#   server
+#     - (miscellaneous changes to server)
+#################################
+
+# Your name.
+author: ChangeMe
```
Comment (Arrow768) [inline html/changelogs/CourierBravo - Lathe_Mats.yml:47]: ChangeMe

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: html/changelogs/CourierBravo - Lathe_Mats.yml (hunk 1 above)
Comment (courierbravo) [reply html/changelogs/CourierBravo - Lathe_Mats.yml]: Fixed the changelog

**Review by DreamySkrell — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**